pkg/etcd/etcd.go

```diff
@@ -419,3 +422,55 @@ func (e *ETCD) removePeer(ctx context.Context, id, address string) error {
 
 	return nil
 }
+
+func (e *ETCD) promoteMember(ctx context.Context, clientAccessInfo *clientaccess.Info) error {
+	clientURLs, _, err := e.clientURLs(ctx, clientAccessInfo)
+	if err != nil {
+		return err
+	}
+	t := time.NewTicker(5 * time.Second)
```

Comment:
Should this be a configurable interval?

Reply:
I am fine with this idea, but setting a configurable interval may cause some issues with slow machines for example, i think its safer to hardcode this value